<!-- required for map merger, known intial poses case-->
<launch>

<group ns="tb3_0/map_merge">
  <param name="init_pose_x" value="-3.2"/>
  <!--param name="init_pose_y" value="-1.0"/-->
  <param name="init_pose_y" value="0.0"/>
  <param name="init_pose_z" value="0.0"/>
  <param name="init_pose_yaw" value="0.0"/>
</group>

</launch>
